src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Customer.xml

```diff
@@ -57,5 +57,11 @@
         <attribute name="user">
             <group>shop:customer:read</group>
         </attribute>
+
+        <attribute name="gender">
+            <group>admin:customer:read</group>
+            <group>shop:customer:read</group>
+            <group>shop:customer:update</group>
```

Comment:
Can we test changes in the gender field? It may be added to the already existing PHPUnit test. Otherwise, let's remove this serialization group